packages/forms-web-app/src/controllers/common/enter-code.js

```diff
@@ -317,13 +352,13 @@ const lpaTokenVerification = (res, token, views, id) => {
  * @param {import('express').Request} req
  * @returns {Promise<void>}
  */
-async function sendTokenToLpaUser(req) {
+const sendTokenToLpaUser = async (req) => {
```

Comment:
Intrigued by this change! I personally prefer explicit functions unless there's a need for arrow function, but we rarely reference `this` where it would make a difference. It seems there's a general preference to use arrow functions though.

Reply:
I'm not that fussed but largely done here for consistency with every other function in this file.
But I'm sold on the idea of const arrow functions by default, can't reassign, no hidden hoisting, no this confusion
less sold on multiple nested anonymous arrow functions

Reply:
Good enough reasons for me!